semgrep-core/tests/yaml/semgrep/ellipses_stress.sgrep

```diff
@@ -0,0 +1,22 @@
+rules:
```

Comment:
This files does not have a corresponding .yaml? Maybe put it in tests/yaml/semgrep/pattern/

Reply:
oh sorry, you actually did that :)